example/src/App.tsx

```diff
@@ -37,30 +37,11 @@ export const App: React.FC = () => {
           <Button title="Next" onPress={next} />
         </ButtonsGroupView>
       </SpotDescriptionView>
-    )
+    ),
   }, {
     alignTo: Align.SCREEN,
     position: Position.BOTTOM,
-    render: () => {
-      // You can also use the hook instead of the props here!
-      const { previous, next } = useSpotlightTour();
```

Comment:
Moved the render function to its own component so the `react-hooks/rules-of-hooks` rule won't complain. The rule doesn't know the function is indeed a function component. We'll solve this later by deprecating the `render` property and changing its name to `Component`. Function which names start with an uppercase are considered components (by the rule, and by a React convention)